rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/helm/HelmBakeManifestRequest.java

```diff
@@ -1,11 +1,14 @@
 package com.netflix.spinnaker.rosco.manifests.helm;
 
+import com.netflix.spinnaker.kork.artifacts.model.Artifact;
 import com.netflix.spinnaker.rosco.manifests.BakeManifestRequest;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 
+import java.util.List;
+
 @Data
 @EqualsAndHashCode(callSuper = true)
 public class HelmBakeManifestRequest extends BakeManifestRequest {
-    String namespace;
```

Comment:
@jorgebee65 We still want namespace here instead of in `BakeManifestRequest` since it applies to Helm but not Kustomize.

Reply:
added namespace, and remove from BakeManifestRequest